.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - '5'
-  - '4'
-  - '0.10'
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
+  - "iojs"
 before_install:
-  - npm install -g npm
+  - if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then npm install -g npm@2; fi
+matrix:
+  fast_finish: true
+cache:
+  directories:
+    - node_modules
```

Comment:
That is true but mostly for packages that use git as a package version.

Everyone from the team has the needed rights to clear the cache if needed so we'll see how it goes.

So far, I haven't seen any issue in any contrib plugin; only the gruntjs.com package, but that is using a package via git.